### PR TITLE
Update nginx-mainline.md

### PR DESCRIPTION
**Pull request by EzequielBruni**
I edited the installation section to remove the whole thing about nginx-mainline. It's really not needed anymore.

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/guides/web/nginx-mainline.md [hunk 1]
```diff
@@ -47,52 +35,38 @@ You’ll need:
...
-To verify that the lastest version of *Nginx* has been installed, run:
+To verify that the latest version of *Nginx* (the latest version from the Rocky repos, anyway) has been installed, run:
 
 ```bash
 nginx -v
 ```
 
 From there, you could just start dropping HTML files into the `/usr/share/nginx/html/` directory to build a simple, static website. The configuration file for the default website/virtual host is called “nginx.conf” and it’s in `/etc/nginx/`. It also holds a number of other basic Nginx server configurations, so even if you choose to move the actual website config to another file, you should probably leave the rest of "nginx.conf" intact.
 
+!!! Note
```
Comment (SergeCroise) [inline docs/guides/web/nginx-mainline.md:66]:
```suggestion
!!! note "Note"
```

**Review by SergeCroise — COMMENTED** [review]
new admonitions with **type "and title"**
would make the work for translators easier 
(feel free to modify or revert my changes).

File: docs/guides/web/nginx-mainline.md [hunk 2]
```diff
@@ -47,52 +35,38 @@ You’ll need:
...
+
+    Old versions of this guide used to detail how to install the nginx-mainline package from ELRepo. This is no longer an option, however. In most cases, the version of Nginx in Rocky's repos is more than sufficient, providing a stable base with backported security patches. Those who still want to use the nginx-mainline branch can follow [this guide from LinuxCapable](https://www.linuxcapable.com/how-to-install-nginx-mainline-on-rocky-linux/). Please note that using nginx-mainline is usually perfectly viable, but not supported.
+
 ## Configuring the Firewall
 
 !!! Note
```
Comment (SergeCroise) [inline docs/guides/web/nginx-mainline.md:72]:
```suggestion
!!! note "Note"
```

Comment (sspencerwire) [reply docs/guides/web/nginx-mainline.md:72]: @SergeCroise  As noted in another PR, I'm not going to convert the regular admonition to a custom admonition. The translators said it was not necessary when we brought this up before.

**Review by sspencerwire — APPROVED** [review]
Thank you for doing this @EzequielBruni !!

**Pull request merged**